src/Tizen.Multimedia.Remoting/WebRTC/MediaFileSource.cs

```diff
@@ -30,8 +29,7 @@ namespace Tizen.Multimedia.Remoting
     /// </remarks>
```

Comment:
As it is written, it looks like both privileges are necessary always. It usually depends on the storage used.
Consider changing like this at the beginning: "Depending on where the source file is located either the media storage privilege (http://tizen.org/privilege/mediastorage) is required or the external storage privilege(http://tizen.org/privilege/externalstorage) is required"

Reply:
OK I'll change it.